events/race.md

```diff
@@ -56,11 +56,35 @@ Indicates that racers should start racing.
 
 Indicates the racer has passed a gate, possibly completed a lap, 
 
-- TODO: Rename this or break it into multiple events? `racer_completed_lap` and `racer_passed_gate`
-- TODO: This doesn't indicate the gate, should it?
+### Properties
+- `fast` [boolean]: Racer's fastest lap
+- `streak` [boolean]: Indicates if the racer is on a streak
+- `valid` [boolean]: Valid gate pass
```

Comment:
I assume this means that this lap can be counted? i.e. When Laps are under the shortest lap time this would be ```"valid":false```

Reply:
Correct. "minimum lap times".

I updated it to have a couple scenarios of when it may be flast.
- `valid` [boolean]: Gate passes can be invalid in some cases such missed gates or minimum lap times